src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 18
+#define SOF_ABI_MINOR 19
```

Comment:
This part not needed since the ABI remains the same.

Reply:
Removed.